python/ray/_private/prometheus_exporter.py

```diff
@@ -293,8 +296,10 @@ def emit(self, view_data):  # pragma: NO COVER
         """
 
         for v_data in view_data:
-            if v_data.tag_value_aggregation_data_map:
-                self.collector.add_view_data(v_data)
+            if v_data.tag_value_aggregation_data_map is None:
```

Comment:
Without this, it will not work because when the worker is dead, `tag_value_aggregation_data_map` can be empty, and without this change, it will be no-op (so we cannot overwrite view_data)